src/externals/exportImage.js

```diff
@@ -0,0 +1,17 @@
+import html2canvas from 'html2canvas'
```

Comment:
I expected this to be Reason :P

Reply:
This component was a bit tricky to rewrite in Reason, so perhaps I'll come back to it. I've spent a bunch of time trying to rewrite it, so maybe you can try?